images/scripts/rhel.setup

```diff
@@ -296,6 +289,14 @@ EOF
     version=$(cut -d '-' -f 2 <<< "$IMAGE")
     su builder -c "/usr/bin/mock --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
     su builder -c "/usr/bin/mock --install --verbose rpmlint"
+
+    # We need to make sure mock has the same selinux-policy as we do
+    if [ "${IMAGE#rhel-9*}" != "$IMAGE" ]; then
+        mkdir /tmp/selinux-policy-rpms
+        dnf download --downloaddir /tmp/selinux-policy-rpms selinux-policy{,-devel,-targeted}
+        dnf install /tmp/selinux-policy-rpms/*.rpm
+        su builder -c "/usr/bin/mock --install /tmp/selinux-policy-rpms/*.rpm"
```

Comment:
Do the host and mock share `/tmp`?  I guess the bots will tell us.  If not, there is `mock --copy-in`.

Reply:
`--install` supports installing from local packages.